barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_internal.hpp

```diff
@@ -292,18 +296,28 @@ template <class DeciderProvingKeys_> class ProtogalaxyProverInternal {
      */
 
     template <size_t skip_count = 0>
-    static void extend_univariates(
-        std::conditional_t<skip_count != 0, ExtendedUnivariates, ExtendedUnivariatesNoOptimisticSkipping>&
+    BB_INLINE static void extend_univariates(
+        std::conditional_t<
```

Comment:
think it would help readability to define this conditional (that now has an inner conditional) separately